Bug: include could not find requested file FindExtPackages

### Description

When i try to import `casbin-cpp` by `FetchContent_xxx` as the description in the `README.md`, i run into an CMake error.

```
[cmake] CMake Error at build/_deps/casbin-src/CMakeLists.txt:88 (include):
[cmake]   include could not find requested file:
[cmake] 
[cmake]     FindExtPackages
[cmake] 
[cmake] 
[cmake] -- [casbin]: Installing casbin ...
[cmake] -- [casbin]: Installing casbin ... -  The targets can now be imported with find_package(casbin)
[cmake] -- [casbin]: Build the "install" target and add "xxxx/install/include" to you PATH for casbin to work
[cmake] -- Configuring incomplete, errors occurred!
```
### Your System Info

Windows 10 Pro 22H2 19045.3208
CMake 3.26.4

### Reason

In casbin-cpp's root directory's  `CMakeLists.txt`, line 17~20:

```cmake
set(CMAKE_MODULE_PATH
    ${CMAKE_MODULE_PATH}
    ${CMAKE_SOURCE_DIR}/cmake/modules
)
```
Is that means i have to manually copy or move the `cmake` folder to my project's root directory just before building my project?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,11 +14,6 @@
 
 cmake_minimum_required(VERSION 3.19)
 
-set(CMAKE_MODULE_PATH
-    ${CMAKE_MODULE_PATH}
-    ${CMAKE_SOURCE_DIR}/cmake/modules
-)
-
 set(CMAKE_WARN_DEPRECATED ON)
 set(PY_CASBIN_VERSION 1.1)
 
@@ -43,6 +38,10 @@ project(
     LANGUAGES CXX C
 )
 
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    ${PROJECT_SOURCE_DIR}/cmake/modules
+)
 
 ###############################################################################
 # Forbid in-source build.

